docs/contributing.md

```diff
@@ -33,6 +33,8 @@ To create a local virtual environment for development, use the [`Python: Create
 
 Once you've set up your virtual environment, ensure you are using the correct intepreter from your local `.venv` directory from the [`Python: Select Interpreter`](https://code.visualstudio.com/docs/python/environments#_select-and-activate-an-environment) command.
 
+When we update the development requirements (especially Airflow itself), you may want to just delete your `.venv` directory and run through these instructions again.
```

Comment:
`just delete your `.venv` directory`, will this affect users' other virtual env?

Reply:
No, the `.venv` directory is local to this repository, assuming the user set it up with VSCode according to the instructions.